apps/sms/js/information.js

```diff
@@ -390,6 +408,7 @@ Information.prototype = {
   show: function() {
     // Hide the Messages edit icon, view container and composer form
     this.parent.classList.add(this.name + '-information');
```

Comment:
Please remove this because the "parent" here is not thread panel anymore and we don't have to controll the stying on new panel(All the styling is related to element on thread header), So please also remove the styling in sms.css (The part under /* Elements related to information view */)